### PR TITLE
Convert applicable INFO logs to WARNING or SEVERE (#1950)

### PR DESCRIPTION
**Pull request by ssoloff**
As mentioned in [#1950](https://github.com/triplea-game/triplea/issues/1950#issuecomment-309639988), there are some `INFO` logs that are actually errors and should have their level increased to either `WARNING` or `SEVERE`.  The purpose of this PR is to identify those instances and increase their level accordingly before proceeding further with fixing #1950.

There are 50 instances of `INFO` logs in the codebase.  These were identified by searching for references to `Logger#info()` and `Logger#log(Level.INFO, ...)`.  The initial commit in this PR simply identifies those instances, and adds a comment immediately before it containing the proposed log level.  I believe most of the logs will remain at the `INFO` level.

Please review and comment on any proposed log levels you do not agree with.  Once an agreement has been reached, I'll update the PR accordingly.

**UPDATE:** All applicable `INFO` logs have been upgraded to either `WARNING` or `SEVERE`.  See [this comment](https://github.com/triplea-game/triplea/pull/2263#issuecomment-324822288) for additional changes that were made.

### CODE REVIEW

**Comment by prastle** [comment]
This is a great idea! @ssoloff

**Comment by ssoloff** [comment]
@prastle Well, it's really @RoiEXLab's and @ron-murhammer's idea.  I'm just executing it. :smile:

**Comment by prastle** [comment]
Sigh i need to try and read more of this here ... :)

File: src/main/java/games/strategy/engine/lobby/server/db/Database.java [hunk 1]
```diff
@@ -260,6 +262,7 @@ private static void shutDownDB() {
         logger.log(Level.WARNING, se.getMessage(), se);
       }
     }
+    // INFO
     logger.info("Databse shut down");
```
Comment (ron-murhammer) [inline src/main/java/games/strategy/engine/lobby/server/db/Database.java]: Typo here

File: src/main/java/games/strategy/engine/lobby/server/db/UserController.java [hunk 2]
```diff
@@ -45,6 +45,7 @@ public HashedPassword getPassword(final String userName) {
       ps.close();
       return returnValue == null ? null : new HashedPassword(returnValue);
     } catch (final SQLException sqle) {
+      // SEVERE
       logger.info("Error for testing user existence:" + userName + " error:" + sqle.getMessage());
```
Comment (ron-murhammer) [inline src/main/java/games/strategy/engine/lobby/server/db/UserController.java]: Probably should change this log message so it isn't a duplicate of the one below.

File: src/main/java/games/strategy/engine/lobby/server/db/UserController.java [hunk 3]
```diff
@@ -176,6 +178,7 @@ public DBUser getUserByName(final String userName) {
       ps.close();
       return user;
     } catch (final SQLException sqle) {
+      // SEVERE
       logger.info("Error for testing user existence:" + userName + " error:" + sqle.getMessage());
```
Comment (ron-murhammer) [inline src/main/java/games/strategy/engine/lobby/server/db/UserController.java]: Probably should change this error message as well.

**Comment by ron-murhammer** [comment]
@ssoloff Added a few comments but pretty much agree with all your choices. Added comments where we should consider improving/fixing the log messages since we are looking at them.

File: src/main/java/games/strategy/engine/lobby/server/db/BadWordController.java [hunk 4]
```diff
@@ -69,6 +70,7 @@ void removeBannedWord(final String word) {
       ps.close();
       return rVal;
     } catch (final SQLException sqle) {
+      // SEVERE
       logger.info("Error reading bad words error:" + sqle.getMessage());
```
Comment (DanVanAtta) [inline src/main/java/games/strategy/engine/lobby/server/db/BadWordController.java]: Maybe remove instead? 
`IllegalStateException(sqle.getMessage());` should probably be: `IllegalStateException(sqle)`;

Logging and throwing indicates to me we are going into an unhandled crash. In that case, the stack trace should be logged. In that case the logging we are doing is redundant, and instead we could attach the extra message and propagate the exception eg:
``` throw new IllegalStateException("Error reading bad words error", sqle)```

Comment (ssoloff) [reply src/main/java/games/strategy/engine/lobby/server/db/BadWordController.java]: :+1: I was already in the process of implementing your first suggestion, but I think your second suggestion is the right thing to do and will update accordingly.

**Comment by ssoloff** [comment]
I think this PR is now ready for review.  Both @ron-murhammer's and @DanVanAtta's suggestions have been implemented.

I also made the following changes:

* Converted calls to `Logger#log(Level.INFO, ...)` to `Logger#info()`.
* Renamed a few `Logger` fields that were not named `logger` simply to ensure consistency across the codebase.

I will update the PR title and description accordingly.

**Pull request merged**